e2e/lvmtest/lvmtest.go

```diff
@@ -0,0 +1,188 @@
+package lvmtest
+
+import (
+	"errors"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+)
+
+var (
+	brickPrefix = "/d/backends/"
+	lvmPrefix   = "patchy_snap"
+	pathPrefix  []string
+)
+
+func verifyLVM() bool {
+	out, err := exec.Command("/sbin/lvcreate", "--help").Output()
+	if err != nil {
+		return false
+	}
+	thinLV := strings.Contains(string(out), "thin")
+	return thinLV
+}
+
+func createBrickPath(num int) ([]string, error) {
+
+	var brickPath []string
+
+	for i := 1; i <= num; i++ {
+		prefix := brickPrefix + strconv.Itoa(i) + "/" + lvmPrefix
+		pathPrefix = append(pathPrefix, prefix)
+		path := prefix + "_mnt"
+		err := os.MkdirAll(path, os.ModeDir|os.ModePerm)
+		if err != nil {
+			return brickPath, err
+		}
+		brickPath = append(brickPath, path)
+	}
+
+	return brickPath, nil
+}
+
+func createLV(num int, thinpoolSize, virtualSize string) error {
+
+	for i := 1; i <= num; i++ {
+		brickPath := pathPrefix[i-1] + "_mnt"
+		devicePath := pathPrefix[i-1] + "_loop"
+		vg := lvmPrefix + "_vg_" + strconv.Itoa(i)
+		poolPath := "/dev/" + vg + "/thinpool"
+		xfsPath := "/dev/" + vg + "/brick_lvm"
+
+		if _, err := exec.Command("/sbin/pvcreate", devicePath).Output(); err != nil {
```

Comment:
all commands can be the constants

```
/sbin/pvcreate
/sbin/vgcreate
/sbin/vgcreate
/usr/sbin/mkfs.xfs
```
please take care of other commands also ,add those as constants